hack/sync_manifests.py

```diff
@@ -0,0 +1,184 @@
+except requests.RequestException as e:
+    print(f"Failed to fetch the latest version: {e}")
+    exit(1)
+
+# Define directories
+bindata_dir = "bindata/assets/kueue-operator"
+manifests_url = f"https://github.com/kubernetes-sigs/kueue/releases/download/v{latest_version}/manifests.yaml"
+input_file = f"manifests.yaml"
+
+# Download manifests.yaml.
+print(f"Downloading manifests.yaml from {manifests_url} ...")
+response = requests.get(manifests_url)
+if response.status_code != 200:
+    raise RuntimeError(f"Failed to download manifests.yaml (HTTP {response.status_code})")
+
+with open(input_file, "wb") as f:
+    f.write(response.content)
+
+# Read and split YAML file.
+try:
+    with open(input_file, "r") as f:
+        docs = list(yaml.safe_load_all(f))
+except yaml.YAMLError as e:
+    print(f"Failed to parse YAML file: {e}")
+    exit(1)
+
+# Define a mapping of 'kind' to filenames.
+file_map = {
+    "CustomResourceDefinition": "crd",
+    "ClusterRole": "clusterrole",
+    "ClusterRoleBinding": "clusterrolebinding",
+    "APIService": "apiservice",
+    "ValidatingWebhookConfiguration": "validatingwebhook",
+    "MutatingWebhookConfiguration": "mutatingwebhook",
+    "Service": "service",
+    "Role": "role",
+    "RoleBinding": "rolebinding",
+    "ServiceAccount": "serviceaccount",
+    "Deployment": "deployment",
+    "Secret": "secret",
+}
+
+# Resources that need namespace updates (excluding Secret)
+namespace_updates = ["Deployment", "Service", "ServiceAccount"]
+
+# Clean up names for RoleBinding, ClusterRoleBinding, Role, Service, and ClusterRole
+def clean_name(name, kind):
+    # Remove 'kueue-' prefix from the name
+    if kind in ["RoleBinding", "ClusterRoleBinding", "Role", "Service", "ClusterRole"]:
+        name = name.replace("kueue-", "")
+    # Remove suffixes for specific kinds
+    if kind in ["RoleBinding", "ClusterRoleBinding"]:
+        name = name.replace("-rolebinding", "").replace("-binding", "")
+    elif kind == "Role":
+        name = name.replace("-role", "")
+    return name
+
+# Organize manifests for output
+separated_manifests = {}
+allowed_kinds = set(file_map.keys())  # Process only these kinds
+
+for doc in docs:
+    if not isinstance(doc, dict) or "kind" not in doc:
+        continue  # Skip invalid or empty YAML docs
+
+    kind = doc["kind"]
+    if kind not in allowed_kinds:
+        continue  # Skip unrelated kinds
+
+    base_filename = file_map[kind]
+
+    # Update namespace for specific resources (excluding Secret)
+    if kind in namespace_updates and "metadata" in doc:
+        doc["metadata"]["namespace"] = "openshift-kueue-operator"
+
+    # Preserve OpenShift-specific label for webhook-service.yaml
+    if kind == "Service" and doc["metadata"]["name"] == "kueue-webhook-service":
+        if "labels" not in doc["metadata"]:
+            doc["metadata"]["labels"] = {}
+        doc["metadata"]["labels"]["service.beta.openshift.io/serving-cert-secret-name"] = "kueue-webhook-server-cert"
```

Comment:
It looks like we don't need https://github.com/openshift/kueue-operator/pull/24/files#diff-64d97c2af009e5921341ae981a3cade926351feb0f38b4d7a24abd44b9703aadL8 anymore. I'm going to remove it from the script.